lib/std/zig/system/windows.zig

```diff
@@ -38,3 +43,228 @@ pub fn detectRuntimeVersion() WindowsVersion {
 
     return @intToEnum(WindowsVersion, version);
 }
+
+const Armv8CpuInfoImpl = struct {
+    cores: [8]*const Target.Cpu.Model = undefined,
+    core_no: usize = 0,
+
+    const cpu_family_models = .{
+        // Family, Model, Revision
+        .{ 8, "D4C", 0, &Target.aarch64.cpu.microsoft_sq3 },
+    };
```

Comment:
Perhaps this list can be merged with https://github.com/ziglang/zig/blob/a0af3c7a19c8f60f740e9a8c178ce19d85b26256/lib/std/zig/system/linux.zig#L175-L217

Currently this list is in a linux-only file but it seems to me that it should be extracted into an arm-only file and then shared by both windows and linux.

Reply:
I was hesitant to do it at first, but forgot there is a subsystem called WSL2 on Win and check it out: https://github.com/hrw/arm-socs-table/issues/15 So yeah, I will rework this as requested. Great suggestion btw!

Reply:
I have extracted the list of CPU models into `system/arm.zig` which is now shared between Linux and Windows CPU detection.